activity_browser/app/bwutils/multilca.py

```diff
@@ -9,18 +9,37 @@
 from .commontasks import wrap_text
 from .metadata import AB_metadata
 
+
 class MLCA(object):
-    # todo: update description
-    # todo: add characterized inventories
     """Wrapper class for performing LCA calculations with many functional units and LCIA methods.
 
-    Needs to be passed a ``calculation_setup`` name.
+    Needs to be passed a brightway ``calculation_setup`` name.
+
+    This class does not subclass the `LCA` class, and performs all
+    calculations upon instantiation.
+
+    Initialization creates `self.lca_scores`, which is a NumPy array
+    of LCA scores, with rows of functional units and columns of LCIA methods.
+    Ordering is the same as in the `calculation_setup`.
 
-    This class does not subclass the `LCA` class, and performs all calculations upon instantiation.
+    Class adapted from bw2calc.multi_lca.MultiLCA to include also
+    CONTRIBUTION ANALYSIS.
```

Comment:
perhaps write something like this "process and elementary flow contributions and other LCA results (see attributes below)"
came across this as we also implement our own contribution analysis class...

Reply:
How about:

> This class is adapted from `bw2calc.multi_lca.MultiLCA` and includes a
    number of additional attributes required to perform process- and
    elementary flow contribution analysis (see class `Contributions` below).